### PR TITLE
Refactor rg-tag

### PR DESCRIPTION
**Pull request by jess0407**
https://rockabox.tpondemand.com/entity/12378
merge with:
https://github.com/rockabox/rbx_campaign_management/pull/427

pending version of rg-tag without action buttons. (for displaying emails and breakdowns)

### CODE REVIEW

File: src/rg-tag/rg-tag.css [hunk 1]
```diff
@@ -54,18 +57,78 @@
...
+    color: var(--color-white);
     background: var(--Tag-hover-color);
+    border: 1px solid var(--Tag-hover-color);
 }
 
 .rgTag button:focus {
     outline: none;
 }
+
+.rgTag button {
```
Comment (nwhite89) [inline src/rg-tag/rg-tag.css]: should use the standard of using ``rgTag--button`` etc.

File: src/rg-tag/rg-tag.css [hunk 2]
```diff
@@ -32,12 +33,14 @@
 .rgTag:hover {
     background: var(--Tag-hover-color);
     text-decoration: none;
+    border: 1px solid var(--Tag-hover-color);
 }
 
 .rgTag--readonly {
     background: var(--Tag-readonly-background-color);
+    border: 1px solid #35353A;
```
Comment (nwhite89) [inline src/rg-tag/rg-tag.css]: ``var (--color-gray-tuna)``

File: src/rg-tag/rg-tag.css [hunk 3]
```diff
@@ -32,12 +33,14 @@
...
     border-radius: 16px;
-    color: var(--Tag-font-color);
+    color: #4D505D;
```
Comment (nwhite89) [inline src/rg-tag/rg-tag.css]: Looks like this should be `var (--color-gray-trout)`

File: src/rg-tag/rg-tag.css [hunk 4]
```diff
@@ -54,18 +57,78 @@
...
 .rgTag button:focus {
     outline: none;
 }
+
+.rgTag button {
+    width: 10px;
+    height: 10px;
+}
+
+.rgTag.rgTag--readonly span.icon {
```
Comment (nwhite89) [inline src/rg-tag/rg-tag.css]: same here these should use the same convention as other Component

File: src/rg-tag/rg-tag.css [hunk 5]
```diff
@@ -54,18 +57,78 @@
...
     background: var(--Tag-hover-color);
+    border: 1px solid var(--Tag-hover-color);
 }
 
 .rgTag button:focus {
     outline: none;
 }
+
+.rgTag button {
+    width: 10px;
```
Comment (nwhite89) [inline src/rg-tag/rg-tag.css]: width and height's should be variables

File: src/rg-tag/rg-tag.css [hunk 6]
```diff
@@ -54,18 +57,78 @@
...
+    height: 10px;
+    display: inline-block;
+    overflow: hidden;
+    text-indent: 200%;
+    white-space: nowrap;
+    background: transparent url("./images/darkPlus.svg") no-repeat 50%;
+}
+
+.rgTag.rgTag--readonly:hover span.icon {
+    width: 10px;
```
Comment (nwhite89) [inline src/rg-tag/rg-tag.css]: bit confused why you need to have this so many times?? are you overriding other styles?

**Comment by darwinlopez** [comment]
@rockabox/qa over to you

**Comment by jim-dj** [comment]
Can be merged for QA

**Pull request merged**